about.html

```diff
@@ -0,0 +1,127 @@
+      <p class="send-email">Please contact us per Email for any further questions about MotoGP Grand Prix 2024!<br>
+        <a href="https://www.motogp.com/en/contact">MotoGP.contact</a></p>
+    </section>
+
+    <section class="event">
+      <div class="event-about">
+        <h2>
+          MotoGP <br> Monster Energy British Grand Prix.
+        </h2>
+        <hr>
+        <p class="motogp-history">The 1949 Grand Prix motorcycle racing season was the inaugural F.I.M. Road Racing World Championship Grand Prix season. The season consisted of six Grand Prix races in five classes: 500cc, 350cc, 250cc, 125cc and Sidecars 600cc. It began on 17 June, with Isle of Man TT and ended with Nations Grand Prix on 4 September.</p>
+      </div>
+      <div class="about-logo">
+        <img src="icons/MotoImg/Moto_Gp_logo.svg.png" alt="motoGP-logo">
+        <div class="about-logo-text">
+          <p>MotoGP</p>
+          <p>British Grand Prix 2023</p>
+        </div>
+      </div>
+    </section>
+
+    <section class="past-races">
+      <h2>See some past races</h2>
+      <hr>
+      <p>Take a look at the past two MotoGP races and read the stories of our students</p>
+
+      <!--<div class="videos">
+         <ul>
+          <li><a href="#">MotoGP Grand Prix<h3>2023</h3></a></li>
+          <li><a href="#">MotoGP Grand Prix<h3>2022</h3></a></li>
+          <li><a href="#">MotoGP Grand Prix<h3>2021</h3></a></li>
+        </ul> 
+      </div>-->
+      <div class="videos">
+        <div class="vid-1">
+          <a href="#">MotoGP Grand Prix<h3>2023</h3></a>
+        </div>
+        <div class="vid-2">
+          <a href="#">MotoGP Grand Prix<h3>2022</h3></a>
+        </div>
+        <div class="vid-3">
+          <a href="#">MotoGP Grand Prix<h3>2021</h3></a>
+        </div>
+      </div>
+    </section>
+
+    <section id="sponsors" class="hidden">
+      <h2>Official Sponsors</h2>
+      <hr>
+      <ul class="sponsor-list">
+        <li><a href="https://www.tissotwatches.com/en-us/">Tissot</a></li>
+        <li><a href="https://www.michelinmotorsport.com/en/motorsport/">Michelin</a></li>
+        <li><a href="https://estrellagalicia00.es/">Estrella Galicia</a></li>
+        <li><a href="https://www.dhl.com/global-en/home.html">DHL</a></li>
+      </ul>
+    </section>
+  </main>
+  <footer class="hidden">
+    <img class="footer-logo" src="icons/MotoImg/Fédération_Internationale_de_Motocyclisme_logo.svg.png" alt="" class="footer-desktop-img">
+    <div>
+      <p>@2023 Dorna Sports SL <br> All rights reserved <br> All trademarks are the property of their respective owners.</p>
+      <p class="privacy-policy hidden">For details about how we use your information, please see our <a href="#">Privacy Policy</a></p>
+      <p class="cookies-terms hidden"><a href="#">Terms & Conditions</a> | <a href="#">Cookies</a></p>
+      </div>
+  </footer>
+<script src="./JS/index.js"></script>
```

Comment:
- Please make sure to stick as much as possible, which includes:
   - [ ] Adjusting the images section on the top of the images below to be on one row in the desktop view to match the given template.
   - [ ] Removing the "Partners" section in the "About" page to reflect the given design.
   - [ ] Changing the background of the footer to match the original design.

| Your design | Original design |
| --- | --- |
| ![image](https://github.com/JohnteFresh/The-Grand-Prix/assets/62290545/4ec7b79e-98b5-4eac-b702-acc773a3e8b4) | ![image](https://github.com/JohnteFresh/The-Grand-Prix/assets/62290545/6dec2e42-d9f1-418b-b2ca-61d2191f2b6a) |

Reply:
Resolved according to the review.